packages/form/core/form-control.ts

```diff
@@ -47,6 +49,7 @@ export class FormControl {
 			label = '',
 			placeholder = null,
 			validators = [],
+			triggerValidationOn = '',
```

Comment:
As @ayoayco said, this `triggerValidationOn` should have a `blur` value as default.